<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>808</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="YCbCrDisplayWidget" name="glwgt_video">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="controls">
      <item>
       <widget class="QPushButton" name="btn_play">
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="SeekBar" name="seek_bar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>996</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="act_file_open"/>
    <addaction name="separator"/>
    <addaction name="act_exit"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="act_about"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="act_file_open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="act_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="act_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>YCbCrDisplayWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>numbat/YCbCrDisplayWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SeekBar</class>
   <extends>QSlider</extends>
   <header>numbat/widgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
